.github/scripts/check-for-update.sh

```diff
@@ -1,178 +0,0 @@
-#!/bin/bash
-
-echo ""
-echo "=========================="
-echo "> df -h -B M"
-df -h -B M
-
-set -eu
-shopt -s inherit_errexit || true
-
-cd .github/slack/
-yarn install
-cd ../..
-
-npm i -g json
-
-branch="$1"
-
-no_negatives() {
-  echo "$(($1 < 0 ? 0 : $1))"
-}
-
-run_sync() {
-  dir="$1"
-  branch="$2"
-  cd "$dir"
-  bash .github/scripts/sync.sh "$branch"
-  echo "synced, exit."
-  exit 0
-}
-
-echo ""
-echo "=========================="
-echo "setting up ssh repo"
-
-mkdir -p ~/.ssh
-echo "$SSH_KEY" > ~/.ssh/id_rsa
-chmod 600 ~/.ssh/id_rsa
-ssh-keyscan github.com >> ~/.ssh/known_hosts
-
-git config --global user.email "prismabots@gmail.com"
-git config --global user.name "Prismo"
-
-git remote add github "git@github.com:$GITHUB_REPOSITORY.git"
-git fetch github "$branch"
-git reset --hard "github/$branch"
-git checkout "github/$branch"
-
-# prepare script: read package.json but ignore workspace package.json files, redwood "web" and "api" package.json file
-pkg="var pkg=require('./package.json'); if (pkg.workspaces || pkg.name == '.prisma/client' || pkg.name == 'web' || pkg.name == 'api') { process.exit(0); }"
-
-# since GH actions are limited to 5 minute cron jobs, just run this continuously for 5 minutes
-minutes=5   # cron job runs each x minutes
-interval=10 # run each x seconds
-i=0
-count=$(((minutes * 60) / interval))
-echo ""
-echo "=========================="
-echo "running loop $count times"
-while [ $i -le $count ]; do
-  # increment to prevent forgetting incrementing, and also prevent overlapping with the next 5-minute job
-  i=$((i + 1))
-  echo ""
-  echo "=========================="
-  echo "run $i"
-
-  start=$(date "+%s")
-
-  dir=$(pwd)
-
-  echo "=========================="
-  echo "updating git checkout"
-  git fetch github "$branch"
-  git reset --hard "github/$branch"
-
-  echo "=========================="
-  echo "getting package version:"
-  v=$(bash .github/scripts/prisma-version.sh "$branch")
-  if [ -z "$v" ]
-  then
-        echo "Prisma version is empty: $v"
-        exit 0
-  fi
-  echo "$v (via Npm)"
-  
-  packages=$(find . -not -path "*/node_modules/*" -type f -name "package.json")
-  echo "$packages" | tr ' ' '\n' | while read -r item; do
-    echo "=========================="
-    echo "checking $item"
- 
-    case "$item" in
-    *"./package.json"* | *".github"* | *"functions/generated/client"*)
```

Comment:
Interesting that we excluded a package.json from a generated Client.
I think we still have one of those, or even a new one.
Might this cause trouble?

Reply:
I think this was the base package.json, pnpm handles that and the two others as well

Reply:
(can this be marked as resolved?)